### ISSUE TITLE
Passing big data between components SDK[V2]

### ISSUE THREAD

**Issue by wafaaabdelhafez** [kubeflow/pipelines#7860]
### What steps did you take

1. Read text data from file on minio, write it to `Outputpath` to pass it to next step
`@component(base_image='wafaaabdelhafez/tensorflow-2')
def load_docs(output_path: OutputPath(List[str])):
    from minio import Minio
    import json
    
    client = Minio()
    obj = client.get_object("mlpipeline", "dataset/model_docs.txt")
    for data in obj:
        content = json.loads(data)
    with open(output_path.path, 'w') as f:
        f.write(content)`
2. Read data from the previous task to process it
`@component(base_image='wafaaabdelhafez/tensorflow-2')
def encode_docs(docs: List[str], output_path: OutputPath(List[str])):
    import json
    import tensorflow as tf
    from keras.preprocessing.text import one_hot

    vocab_size = 50000
    encoded_docs = []
    for d in docs:
        res = one_hot(d, vocab_size)
        encoded_docs.append(res)
    with open(output_path, 'w') as f:
        f.write(json.dumps(encoded_docs))` 

I got an error in the first step (reading data and write to outputpath) is it's exceed limits , data too long
### What happened:
cannot write data to `Outputhpath `
### What did you expect to happen:
can read and write big amount of data and pass it through steps of pipeline link in V1.x
### Environment:
Kubeflow 1.4
python 3.7
working on kubeflow notebook

*  How do you deploy Kubeflow Pipelines (KFP)? 

*  KFP version: 

*  KFP SDK version: 

### Labels

/area sdk

---

Impacted by this bug? Give it a 👍. We prioritise the issues with the most 👍.

**Comment by Ark-kun** [comment]
Are you sure you've included the whole code?

I'm not sure this can work: `output_path.path`.
You should replace it with just `output_path`.

Your second issue is that your 2nd component is getting the texts without `InputPath`.
Replace `docs: List[str]` with `docs_path: InputPath(List[str])`.

Then it should work.

Under the hood, in KFP, the way the upstream output is compiled depends on how it's consumed downstream.
In general, outputs using `OutputPath` should be passed to inputs using `InputPath`.

**Comment by rimolive** [comment]
Closing this issue. No activity for more than a year.

/close
